sentiment.py

```diff
@@ -36,8 +36,6 @@ def get_emotion(entry):
             tones.append("Bored")
 
     # correcting the 'fear' to 'fearful' for a better sound
-    for tone in tones:
-        if tone == "Fear":
-            tone = "Fearful"
+    tones = [tone.replace("Fear", "Fearful") for tone in tones]
```

Comment:
I made a mistake earlier, the previous function did not actually work. My bad. This new one line function is working though.